sale_order_route/README.rst

```diff
@@ -0,0 +1,18 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Sale Order Route
+================
+
+This module adds new route_id field in sale order object.
+When the value of the field is changed, all routes from order lines will be filled with this value.
```

Comment:
This module links a sale order with a inventory route. Changing this field will change the route in each of the lines, filling with the value of the order.